registry/json.go

```diff
@@ -21,3 +28,41 @@ func (registry *Registry) getJson(url string, response interface{}) error {
 
 	return nil
 }
+
+// getPaginatedJson accepts a string and a pointer, and returns the
+// next page URL while updating pointed-to variable with a parsed JSON
+// value. When there are no more pages it returns `ErrNoMorePages`.
+func (registry *Registry) getPaginatedJson(url string, response interface{}) (string, error) {
+	resp, err := registry.Client.Get(url)
+	if resp != nil {
```

Comment:
Sure; I had uncritically copied it from the existing function getJSON. I can fix both up.